### PR TITLE
Fixed bugs based on UX feedbacks

### PR DESCRIPTION
**Pull request by ghost**
Please check SCRD-2138 for details.

### CODE REVIEW

File: src/styles/common.less [hunk 1]
```diff
@@ -196,10 +196,10 @@ p {
 }
 
 #helptext > .tooltip-inner {
```
Comment (jgravel) [inline src/styles/common.less]: This whole section (line 198 to 205) can be gone, since black is the default color.

**Review by jgravel — CHANGES_REQUESTED** [review]
- There should be a check mark icon inside the selected model card of the Choose Cloud Model page.
- There should be a check mark icon inside the selected component card of the Cloud Model to Deploy page.
- The import from CSV link on the assign server role page should be below the Add Server button (per her email).

File: src/styles/common.less (hunk 1 above)
Comment (guglo) [reply src/styles/common.less]: @jgravel  thanks for the review. The requirement didn't include the check mark in the Jira item. I will double check with @jeremy-moffitt . 
Regarding csv link. I asked @jeremy-moffitt if it is ok to do side by side...it looks better that way. @jeremy-moffitt  oked. 
ok will fix the tooltip part.

Comment (jeremy-moffitt) [reply src/styles/common.less:198]: I didnt feel strongly about it, though Julie is correct that the UX mock shows the link underneath, I'd go ahead and put it under if you can

Comment (ghost) [reply src/styles/common.less:198]: @jeremy-moffitt @jgravel  ok will add check mark and move the link button under.

**Comment by GarySmith** [comment]
Please apply submit this as a pull request into the new repo at ArdanaCLM/ardana-installer-ui

**Pull request closed without merging**